.github/workflows/flaky-test-monitor.yml

```diff
@@ -0,0 +1,49 @@
+name: Flaky Test Monitor
+
+on:
+  schedule:
+    - cron: '*/30 * * * *' # every 30 minutes
```

Comment:
does it need to run every 30 minutes 24/7? What about overnight hours and weekends? running 24 times/day 24/7/365 seems excessive